server/graphql-engine.cabal

```diff
@@ -90,6 +90,7 @@ library
 
                      -- URL parser related
                      , network-uri
+                     , http-api-data >= 0.4
```

Comment:
maybe slightly overkill for using an instance defined there.

Reply:
yeah, i thought this might be too much.